h2o-py/tests/testdir_misc/pyunit_pivot_medium.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+
+def pivot():
+    df = h2o.create_frame(rows=1000000,
+                          cols=3,
+                          factors=10,
+                          categorical_fraction=1.0/3,
+                          time_fraction=1.0/3,
+                          real_fraction=1.0/3,
+                          real_range=100,
+                          missing_fraction=0.0,
+                          seed=123)
+    dcol = df.moment(year=df["C1"].year(),month=df["C1"].month(),day=df["C1"].day())
```

Comment:
Good you are making a date here. But how would this do if the column is already a date? Assuming the user would have to call `moments` to make a time column H2O can handle?